manifests/pod.yaml

```diff
@@ -83,6 +95,23 @@ spec:
     secret:
       secretName: aws-fcos-kola-bot-config
       optional: true
+  # Necessary secrets for Brew
```

Comment:
For secrets, we've been trying to go in the reverse direction actually and drain the secrets from there. The latest in that area is https://github.com/coreos/fedora-coreos-pipeline/pull/607. You can see an example of how it can be used in https://github.com/coreos/fedora-coreos-pipeline/pull/614. So concretely, the idea is:
- you create the secrets in OpenShift in a format compatible with kubernetes-credentials-provider (see examples here: https://github.com/jenkinsci/kubernetes-credentials-provider-plugin/tree/master/docs/examples)
- they get automatically synced as Jenkins credentials
- in jobs which have to tolerate them not existing, you use `tryWithCredentials()`
- in jobs which can assume they always exist, you use `withCredentials()` (like I think the new dedicated Brew job would)

Reply:
Great! I will remove this part here so. And add necessary stuff in  https://github.com/coreos/fedora-coreos-pipeline/pull/602